ts/src/pro/bitmart.ts

```diff
@@ -542,7 +542,8 @@ export default class bitmart extends bitmartRest {
         //     { event: 'login' }
         //
         const messageHash = 'authenticated';
-        client.resolve (message, messageHash);
+        const future = this.safeValue (client.futures, messageHash);
```

Comment:
@pcriadoperez Why would we replace a one-liner with 2 lines that do the same here?

Reply:
In this case because client.resolve, deletes the future, and we don't want to delete. with @carlosmiei , we decided it's better to manually resolve like this